force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="slds-grid slds-theme_default slds-grid--align-spread">
+
+        <div class="slds-col slds-m-left_x-small">
+            <span>ComponentTitle</span>
```

Comment:
This should be a variable so it is inherited
```suggestion
            <span>{componentTitle}</span>
```